core/src/test/java/org/infinispan/stream/BaseStreamTest.java

```diff
@@ -219,7 +218,7 @@ public void testObjSortedCollector() {
 
       List<Map.Entry<Integer, String>> list = createStream(entrySet).sorted(
             (e1, e2) -> Integer.compare(e1.getKey(), e2.getKey())).collect(
-            CacheCollectors.serializableCollector(Collectors::toList));
+            () -> Collectors.toList());
```

Comment:
`Collectors::toList` should still work right?

Reply:
It doesn't seem to:

    Error:(220, 75) java: incompatible types: cannot infer type-variable(s) R1,A,capture#1 of ?,T
    (argument mismatch; bad return type in method reference
      java.util.stream.Collector<T,capture#1 of ?,java.util.List<T>> cannot be converted to 
      java.util.stream.Collector<? super java.util.Map.Entry<java.lang.Integer,java.lang.String>,A,R1>)

It does work with the fix above... WDYT?